content/en/serverless/installation/nodejs.md

```diff
@@ -17,7 +17,7 @@ After you have [installed the AWS integration][1], choose one of the following m
 {{< tabs >}}
 {{% tab "Serverless Framework" %}}
 
-Use the [Datadog Serverless Plugin][1] to send metrics, logs and traces from your application to Datadog without any code changes. The plugin automatically attaches the Datadog Lambda Library for Node.js to your functions using layers. At deploy time, it swaps your original function handler with the Datadog-provided handler that initializes the Datadog Lambda Library and invokes your original function handler. It also enables Datadog and X-Ray tracing for your Lambda functions.
+The [Datadog Serverless Plugin][1] automatically adds the Datadog Lambda library to your functions using layers, and configures your functions to send metrics, traces and logs to Datadog.
```

Comment:
```suggestion
The [Datadog Serverless Plugin][1] automatically adds the Datadog Lambda library to your functions using layers, and configures your functions to send metrics, traces, and logs to Datadog.
```
Also: "using layers" ... do you mean "adds the library by using layers" or "to your functions that use layers"? or something else?

Reply:
ah, good catch, i meant "by" using layers.